src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.java

```diff
@@ -161,10 +164,15 @@ public void verplaatsEnkelvoudigInformatieobject(final RESTDocumentVerplaatsGege
         final EnkelvoudigInformatieobject informatieobject = drcClientService.readEnkelvoudigInformatieobject(documentUUID);
         final Zaak nieuweZaak = zrcClientService.readZaakByID(documentVerplaatsGegevens.nieuweZaakID);
         if ("ontkoppelde-documenten".equals(documentVerplaatsGegevens.zaakID)) {
-            OntkoppeldDocument ontkoppeldDocument = ontkoppeldeDocumentenService.readByDocumentUUID(documentUUID);
+            final OntkoppeldDocument ontkoppeldDocument = ontkoppeldeDocumentenService.readByDocumentUUID(documentUUID);
             String toelichting = "Verplaatst: %s -> %s".formatted(documentVerplaatsGegevens.zaakID, nieuweZaak.getIdentificatie());
             zrcClientService.koppelInformatieobject(informatieobject, nieuweZaak, toelichting);
             ontkoppeldeDocumentenService.delete(ontkoppeldDocument.getId());
+        } else if ("inbox-documenten".equals(documentVerplaatsGegevens.zaakID)) {
```

Comment:
De string "inbox-documenten" graag als constante definieren en wellicht wat commentaar erbij wat de betekenis is. Want het is op het eerste gezicht vreemd dat die waarde in zaakId zit. Misschien is het ook beter dit veld niet te herbruiken? Geld ook voor  "ontkoppelde-documenten" hierboven

Reply:
zaakid hernoemd naar bron en 2 methodes aangemaakt op verplaatsgegevens vanuitInboxdocumenten en vanuitOntkoppeldedocumenten